server/src/main/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImpl.java

```diff
@@ -1598,11 +1600,13 @@ protected UserVm importUnmanagedInstanceFromVmwareToKvm(DataCenter zone, Cluster
         String ovfTemplateOnConvertLocation = null;
         try {
             HostVO convertHost = selectInstanceConversionKVMHostInCluster(destinationCluster, convertInstanceHostId);
+            HostVO importHost = convertInstanceHostId == null ? convertHost : selectInstanceConversionKVMHostInCluster(destinationCluster, null);
```

Comment:
convert and import host need not be the same, in case of same cluster (can take import host from input? if not provided, let it choose the import host).

keep separate methods for conversion host and import host selection. also, better have disabled host used for conversion (not for import - as it needs resource allocation).